scripts/test/wallet/many-addrs-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+keydir="./dev-keys"
+key1="$keydir/1.key"
+key2="$keydir/2.key"
+addrsNum=1000
+
+echo -e "\nLooking for key files $key1 and $key2"
+
+echo -e "\nResetting wallet database..."
+curl -k https://127.0.0.1:8090/api/test/reset -d ""
+
+
+echo -e "\nCreating 2 wallets..."
```

Comment:
maybe we should run two backends and send money between two nodes (launched on the same machine).

This would be more realistic test of bittrex environment then to send between two wallets within same node/wallet/wallet-db .

Reply:
And I don't envy that guy who will need such scenario for true similarity)